modules/pytket-honeywell/pytket/extensions/honeywell/backends/honeywell.py

```diff
@@ -352,8 +352,7 @@ def _update_cache_result(self, handle: ResultHandle, res: BackendResult) -> None
     def circuit_status(self, handle: ResultHandle) -> CircuitStatus:
         self._check_handle_type(handle)
         jobid = str(handle[0])
-        ppcirc_rep = literal_eval(cast(str, handle[1]))
-        ppcirc = Circuit.from_dict(ppcirc_rep) if ppcirc_rep is not None else None
+        ppcirc_rep = json.loads(cast(str, handle[1]))
```

Comment:
this load only needs to happen after completion too right?

Reply:
Oops, yes.